c++/WORKSPACE

```diff
@@ -45,7 +45,7 @@ cc_library(
 http_archive(
     name = "zlib",
     build_file_content = _zlib_build,
-    sha256 = "d14c38e313afc35a9a8760dadf26042f51ea0f5d154b0630a31da0540107fb98",
+    sha256 = "b3a24de97a8fdbc835b9833169501030b8977031bcb54b3b3ac13740f846ab30",
     strip_prefix = "zlib-1.2.13",
-    urls = ["https://zlib.net/zlib-1.2.13.tar.xz"],
+    urls = ["https://zlib.net/fossils/zlib-1.2.13.tar.gz"],
```

Comment:
This particular diff is not required to build at present, since 1.2.13 is still the current zlib version. I initially added this diff when working off an earlier commit, and should perhaps remove it from this PR.